### ISSUE TITLE
Clone and Manage Bar Buttons

### ISSUE THREAD

**Issue by Marcnesium87** [wol-soft/beerplop-issues#50]
Can we get some Clone and Manage Bar Buttons in the Drop Down Menu, where the Disable/Enable Autobuyer and Level Up Buttons are?
I dont like to scroll Up and Down to see if the Cloning is ready!

**Comment by wol-soft** [comment]
Hi @Marcnesium87 Would this be still required if #40 is implemented? I can see your point that it's too easy to overlook a possible cloning.

**Comment by Marcnesium87** [comment]
Maybe yes, because if you want to Clone a Specific Building (I clone the same Building as fast as I can multiple Times, so the % dont drop down later and the Multiplier rise all in the same Time)
So I have to use a Chrome Stop Time Plugin, to remind me when the Cloning Timer is Zero. 
Even with the Clone Automation, i want to Disable the Automation and clone Dispensers three Times in a row on a fresh Sacrifice.

**Comment by wol-soft** [comment]
Yeah, good point, also interesting when the cloning automation isn't yet unlocked.
